persist.class.nut

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) 2015 Electric Imp
+// This file is licensed under the MIT License
+// http://opensource.org/licenses/MIT
+
+class Persist {
+    static version = [1,0,0];
+
+    _cache = null;  // A local cache of the data
+
+    // Class ctor - creates object and loads data
+    constructor() {
+        _cache = server.load();
+    }
+
+    // Reads a value from the cache
+    // Parameters:
+    //      key:    The key of the data we're requesting
+    //
+    // Returns:     The value (if the key exists)
+    //              null (if the key doesn't exist)
+    function read(key) {
+        return (key in _cache) ? _cache[key] : null;
```

Comment:
I think this should throw an error if key is not in the table. Variables should always be init'd. Convince me otherwise.